modules/order/src/Form/OrderReassignForm.php

```diff
@@ -22,20 +24,33 @@ class OrderReassignForm extends FormBase {
   protected $order;
 
   /**
+   * The order assignment service.
+   *
+   * @var \Drupal\commerce_order\OrderAssignment
+   */
+  protected $order_assignment_service;
```

Comment:
1) You can't use snake_case in a class property
2) We never append _service to any object. You want $orderAssignment.